web/src/components/App.tsx

```diff
@@ -41,19 +44,24 @@ class App extends React.Component<Props, {}> {
 
   componentDidMount() {
     if (RemoteData.isSuccess(this.props.services)) {
-      this.props.fetchErrors(this.props.match.params.service)
+      if (this.props.match.params.service !== "targets") {
+        this.props.fetchErrors(this.props.match.params.service)
+      }
+      this.props.fetchTargets()
```

Comment:
I think we should move the fetching of targets to the TargetList component, we don't really need it here. You could use `componentDidMount` there, or even better, `useEffect` which is the new react way.

I think we should also rename this component to something like "Errors" to avoid confusion and potentially tweak the routes to prepend errors in the route for the Errors page, something like `[/errors/:service/:errorKey", "/errors/:service"]`